### ISSUE TITLE
Inconsistent use of \X{ft}^n and \X{ft}^\ast in execution spec?

### ISSUE THREAD

**Issue by tlively** [WebAssembly/gc#400]
For example, the execution semantics for `struct.new` and `struct.new_default` use `ft^*` in the prose and `ft^n` in the formalism, whereas `struct.get` and `struct.set` use `ft^*` throughout.

Is the rule that if the length matters, use `ft^n` and otherwise use `ft^*`? If so, is it intentional that the prose uses `ft^*` in either case?

**Comment by rossberg** [comment]
Yes, that's the idea. I believe we never use `x^n` in the prose, because we always talk about the length of a sequence explicitly.
